src/features/click-to-load.js

```diff
@@ -1806,7 +1800,20 @@ export default class ClickToLoad extends ContentFeature {
         // Request the current state of Click to Load from the platform.
         // Note: When the response is received, the response handler resolves
         //       the readyToDisplayPlaceholders Promise.
-        sendMessage('getClickToLoadState')
+        const clickToLoadState = await this.messaging.request('getClickToLoadState')
+        this.onClickToLoadState(clickToLoadState)
+
+        // Listen to message from Platform letting CTL know that we're ready to
+        // replace elements in the page
+        // eslint-disable-next-line promise/prefer-await-to-then
+        this.messaging.subscribe(
+            'displayClickToLoadPlaceholders',
+            // TODO: Pass `message.options.ruleAction` through, that way only
+            //       content corresponding to the entity for that ruleAction need to
+            //       be replaced with a placeholder.
+            (_) => replaceClickToLoadElements()
+        )
+
         await readyToDisplayPlaceholders
```

Comment:
@kzar Considering that `await this.messaging.request('getClickToLoadState')` is now pausing this function, and that the `clickToLoadState()` that is called right after is what resolves `readyToDisplayPlaceholders`, I can remove this line, right? 
I noticed we still need this for `replaceClickToLoadElements()`, so I moved its subscribe up so its ready before we pause execution here, in case we have a race condition and get messages from the platform before we got the CTL state response.